HackIllinois/ViewControllers/HIEventListViewController.swift

```diff
@@ -99,19 +102,19 @@ extension HIEventListViewController: HIEventCellDelegate {
 }
 
 // MARK: - UIViewControllerPreviewingDelegate
-extension HIEventListViewController: UIViewControllerPreviewingDelegate {
-    func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
-        guard let tableView = tableView,
-            let indexPath = tableView.indexPathForRow(at: location),
-            let event = _fetchedResultsController?.object(at: indexPath) as? Event else {
-                return nil
-        }
-        previewingContext.sourceRect = tableView.rectForRow(at: indexPath)
-        eventDetailViewController.event = event
-        return eventDetailViewController
-    }
-
-    func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
-        self.present(viewControllerToCommit, animated: true, completion: nil)
-    }
-}
+//extension HIEventListViewController: UIViewControllerPreviewingDelegate {
+//    func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
+//        guard let tableView = tableView,
+//            let indexPath = tableView.indexPathForRow(at: location),
+//            let event = _fetchedResultsController?.object(at: indexPath) as? Event else {
+//                return nil
+//        }
+//        previewingContext.sourceRect = tableView.rectForRow(at: indexPath)
+//        eventDetailViewController.event = event
+//        return eventDetailViewController
+//    }
+//
+//    func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
+//        self.present(viewControllerToCommit, animated: true, completion: nil)
+//    }
+//}
```

Comment:
Add a note of what this used to do?

Reply:
The note stating that event detail view controller will not be used is at the top of the HIEventListViewController.swift file.